setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-install_requires = ["execnet>=1.1", "psutil>=3.0.0", "pytest>=6.0.0", "pytest-forked"]
+install_requires = ["execnet>=1.1", "pytest>=6.0.0", "pytest-forked"]
```

Comment:
we should add a extra for psutil

Reply:
If we add psutil as an extra, does that imply that `auto_detect_cpus`/`pytest_xdist_auto_num_workers` also need to try/except on `psutil`, correct?